examples/jaeger-native-tracing/Dockerfile-frontenvoy

```diff
@@ -0,0 +1,18 @@
+## Image with pre-built Jaeger tracer extension.
+## See https://github.com/yskopets/envoy-jaeger-tracer
+FROM yskopets/envoy-jaeger-tracer:0.4.2-1.13.0-0 AS jaeger-tracer
```

Comment:
Seems like jaeger-tracer latest release is https://github.com/jaegertracing/jaeger-client-cpp/releases/tag/v0.5.0. Do you think you can provide the patch for it and make it "blessed" by upstream?

Reply:
Sure, I will look into it